src/zoo.ml

```diff
@@ -197,7 +196,8 @@ struct
         | None -> fatal_error "I am sorry but this language has no interactive toplevel."
       in
       Format.printf "%s -- programming languages zoo@\n" L.name ;
-      Format.printf "Type %s to exit@." eof ;
+      Format.printf "Type %s to exit@\n" eof ;
```

Comment:
We could, but it is relevant to the toplevel. What about putting it in the main README?